### ISSUE TITLE
Default Preferences

### ISSUE THREAD

**Issue by Bk8** [tableau/tableau-log-viewer#120]
The highlight filter and opened files should remain the same for the next session. This is how other log viewers work and they save a lot of time. Thanks for the software!

**Comment by luiseeo** [comment]
This is a great suggestion!

Remembering the opened files + their corresponding filters would be very close to "Restore last session" functionality

**Comment by Bk8** [comment]
Yes thanks for the logg viewer I love it! Hope you add this feature! Thanks
